src/libraries/System.Private.CoreLib/src/System/Convert.cs

```diff
@@ -104,6 +105,7 @@ public static partial class Convert
         internal const string Base64Table = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
 
         private const int Base64LineBreakPosition = 76;
+        private const int Base64VectorizationLengthThreshold = 64;
```

Comment:
You might want to make it lower, initially it was based on an not as fast as yours SIMD path